VimeoNetworking/VimeoNetworking/VimeoClient.swift

```diff
@@ -55,7 +55,10 @@ final public class VimeoClient
     }
     
         /// Dictionary containing URL parameters for a request
-    public typealias RequestParameters = [String: AnyObject]
+    public typealias RequestDictionary = [String: AnyObject]
+    
+        /// Array containing URL parameters for a request
+    public typealias RequestArray = [AnyObject]
```

Comment:
Can this type be more communicative in its name. Sounds like an array of requests. Maybe something like RequestBodyArray